twilio/rest/accounts/v1/credential/aws.py

```diff
@@ -116,13 +112,13 @@ def get_page(self, target_url):
     def create(self, credentials, friendly_name=values.unset,
                account_sid=values.unset):
         """
-        Create a new AwsInstance
+        Create the AwsInstance
 
         :param unicode credentials: A string that contains the AWS access credentials in the format <AWS_ACCESS_KEY_ID>:<AWS_SECRET_ACCESS_KEY>
         :param unicode friendly_name: A string to describe the resource
         :param unicode account_sid: The Subaccount this Credential should be associated with.
 
-        :returns: Newly created AwsInstance
+        :returns: The created AwsInstance
```

Comment:
Was this intentional?

Reply:
Kind of... side effect of refactoring yoyodyne to reduce duplications.